Sources/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -156,6 +156,41 @@ class SubscriberAttributesManager {
         setReservedAttribute(.creative, value: creative, appUserID: appUserID)
     }
 
+    func setAppsFlyerAttributionData(_ data: [AnyHashable: Any]?, appUserID: String) {
+        guard let data = data else {
+            return
+        }
+
+        let mediaSource = stringValue(from: data, forKey: "media_source")
+            ?? (stringValue(from: data, forKey: "af_status")?.caseInsensitiveCompare("Organic") == .orderedSame
+                ? "Organic" : nil)
+        if let mediaSource = mediaSource {
+            setReservedAttribute(.mediaSource, value: mediaSource, appUserID: appUserID)
+        }
+
+        if let campaign = stringValue(from: data, forKey: "campaign") {
+            setReservedAttribute(.campaign, value: campaign, appUserID: appUserID)
+        }
+
+        if let adGroup = stringValue(from: data, forKey: "adgroup") ?? stringValue(from: data, forKey: "adset") {
+            setReservedAttribute(.adGroup, value: adGroup, appUserID: appUserID)
+        }
+
+        // swiftlint:disable:next identifier_name
+        if let ad = stringValue(from: data, forKey: "af_ad") ?? stringValue(from: data, forKey: "ad_id") {
+            setReservedAttribute(.ad, value: ad, appUserID: appUserID)
+        }
+
+        if let keyword = stringValue(from: data, forKey: "af_keywords") ?? stringValue(from: data, forKey: "keyword") {
+            setReservedAttribute(.keyword, value: keyword, appUserID: appUserID)
+        }
+
+        if let creative = stringValue(from: data, forKey: "creative")
+            ?? stringValue(from: data, forKey: "af_creative") {
```

Comment:
Just curious, for af_ad and af_keywords, we prioritize those keys over the other parameter, but here we prioritize `creative` over `af_creative`. Just making sure that's a conscious decission :)